scripts/release-jetty.sh

```diff
@@ -158,7 +158,9 @@ if proceedyn "Are you sure you want to release using above? (y/N)" n; then
     # This is equivalent to 'mvn release:perform'
     if proceedyn "Build/Deploy from tag $TAG_NAME? (Y/n)" y; then
         git checkout $TAG_NAME
-        mvn clean package source:jar javadoc:jar gpg:sign deploy \
+        # javadoc:aggregate-jar need already build modules
+        mvn clean install -T4 -DskipTests
+        mvn javadoc:aggregate-jar package source:jar javadoc:jar gpg:sign deploy \
```

Comment:
This will definitely fail, a lot.
Can we not just have a single maven step?

Reply:
also, what's the artifact this makes?

Reply:
correct this will fail. I will fix that
The output will be a big jar will all the javadoc so deploying website javadoc after release will be only about download this big jar and unzip it

Reply:
should be better now
I tried locally running on a tag with ` mvn clean  package source:jar javadoc:jar gpg:sign -Peclipse-release -DskipTests javadoc:aggregate-jar install ` so install instead of deploy and it pass.